src/lib/Form/Type/Trash/TrashItemChoiceType.php

```diff
@@ -32,9 +47,14 @@ public function __construct(TrashService $trashService)
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'choices' => $this->trashService->findTrashItems(new Query([
+            'choices' => array_map(function ($item) {
```

Comment:
* add type hinting form `$item` to reduce warnings in IDE
* extract `array_map` call to separate method for better code readability